sdl2/test/sdl2ext_draw_test.py

```diff
@@ -17,7 +17,6 @@
 @pytest.fixture
 def testsurf(with_sdl):
     sf = _create_surface((10, 10), fmt="RGBA32")
-    assert SDL_GetError() == b""
```

Comment:
It's actually redundant here, since  the `_create_surface` convenience function already checks if `sf` is null and raises a Python exception if it is. Guess I didn't think it through when I originally wrote the fixture.

Reply:
OK, that makes sense!